lessons/en/basics/functions.md

```diff
@@ -194,14 +194,14 @@ iex> fred = %{
 %{name: person_name}
 ```
 
-In `Greeter1.hello/1`, the map we pass (`fred`) is evaluated against our argument (`%{name: person_name}`):
+In `Greeter1.hello/1`, the map we pass (`fred`) is evaluated against function argument (`%{name: person_name}`):
```

Comment:
I think the previous text was sufficient but if we want to make it clearer we need to correct the grammar in this change:

```suggestion
In `Greeter1.hello/1`, the map we pass (`fred`) is evaluated against our function argument (`%{name: person_name}`):
```